src/Features/Core/Portable/CodeFixes/Suppression/AbstractSuppressionCodeFixProvider.cs

```diff
@@ -195,7 +195,7 @@ internal class SuppressionTargetInfo
             var syntaxFacts = document.GetLanguageService<ISyntaxFactsService>();
 
             ISymbol targetSymbol = null;
-            var targetMemberNode = syntaxFacts.GetContainingMemberDeclaration(root, startToken.SpanStart);
+            var targetMemberNode = syntaxFacts.GetContainingMemberDeclaration(root, nodeWithTokens.SpanStart);
```

Comment:
`startToken` need not be the first token for `nodeWithTokens`, which is a first common node containing start and end tokens. For example, for the test case for this bug:

```c#
enum E
{
  F1, // some comment
  F2
}
```

`startToken` = `F1` and `endToken` = `,` token and closest node containing both tokens is the enum declaration which has a different span start.